packetbeat/_meta/fields.yml

```diff
@@ -2044,3 +2044,17 @@
           type: keyword
           description: >
             An array containing the TLS alert type for every alert received.
+
+        - name: ja3_fingerprint
+          type: group
+          fields:
+
+            - name: hash
+              type: keyword
+              description: >
+                The JA3 fingerprint hash for the client side.
+
+            - name: str
```

Comment:
I'm on the fence about the field naming. Here are some alternatives I am considering.

`tls.fingerprint.ja3.hash`
`tls.fingerprint.ja3.raw`

`tls.ja3_fingerprint.hash`
`tls.ja3_fingerprint.raw`

`tls.ja3.hash`
`tls.ja3.raw`

// cc: @ruflin WDYT?

Reply:
I originally went for "tls.fingerprint.ja3.hash" and at the last moment changed my mind. Will revert back to it, I think it makes more sense now